src/client/dfs/duns.c

```diff
@@ -676,8 +676,6 @@ create_cont(daos_handle_t poh, struct duns_attr_t *attrp, bool create_with_label
 		if (create_with_label)
 			rc = dfs_cont_create_with_label(poh, attrp->da_cont, &dfs_attr,
 							&attrp->da_cuuid, NULL, NULL);
-		else if (!uuid_is_null(attrp->da_cuuid))
```

Comment:
Did you include some extra fixes to this pr, eg:
https://github.com/daos-stack/daos/pull/9364/files

For backport, normally we don't do this, we just keep as much as possible with master, and port extra
fixes one by one to release/2.2 branch eg.

Reply:
Thanks! There is no issue (PR 9364) in 2.2 as in master. I will port patches one by one to 2.2 in future.